mava/wrappers/jumanji.py

```diff
@@ -266,14 +289,17 @@ def aggregate_rewards(
         # The episode is won if all agents have connected.
         extras = timestep.extras | {"won_episode": timestep.extras["ratio_connections"] == 1.0}
 
-        return timestep.replace(observation=Observation(**obs_data), extras=extras)
+        # Whether or not aggregate the list of individual rewards.
+        reward = aggregate_rewards(timestep.reward, self.num_agents, self._use_individual_rewards)
```

Comment:
Same about the`not` as in my previous comment. 
```suggestion
        if not self._use_individual_rewards:
            reward = aggregate_rewards(timestep.reward, self.num_agents, self._use_individual_rewards)
```